### PR TITLE
Change exported functions signature

### PR DESCRIPTION
**Pull request by mlevieux**
Remove pointers in returns and change function names.

### CODE REVIEW

File: identity/app_secret.go [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
-import (
-	identity2 "github.com/TankerHQ/identity-go/v3"
-)
-
 // DEPRECATED: use github.com/TankerHQ/identity-go/v3
 
-const AppSecretSize = identity2.AppSecretSize
-const AppPublicKeySize = identity2.AppPublicKeySize
+const AppSecretSize = 64
+const AppPublicKeySize = 32
```
Comment (blastrock) [inline identity/app_secret.go:6]: should these be exposed?

File: identity/generate.go [hunk 2]
```diff
@@ -4,28 +4,32 @@ import (
 	identity2 "github.com/TankerHQ/identity-go/v3"
```
Comment (blastrock) [inline identity/generate.go:4]: I don't understand what this is. Isn't it this package itself?

**Review by blastrock — COMMENTED** [review]
I see here that we have put the effort to make compatibility functions to migrate from previous SDKs to v3, but this time we break the API... I guess if we don't have customers using this, it's ok ^^

File: identity/app_secret.go (hunk 1 above)
Comment (mlevieux) [reply identity/app_secret.go:6]: I think they were before, and this folder has been left here only so as to keep the new version compatible with previous ones... So I'd say until we switch major, we need to leave this here :/

File: identity.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ type publicIdentity struct {
 	Value        string `json:"value"`
 }
 
-type identity struct {
+type Identity struct {
```
[the remark this answers is not in the document]
Comment (mlevieux) [reply identity.go:22]: Gonna change this, don't know why I exported them :thinking:

**Pull request closed without merging**